lib/bugsnag/middleware/discard_error_class.rb

```diff
@@ -0,0 +1,30 @@
+module Bugsnag::Middleware
+  ##
+  # Determines if the exception should be ignored based on the configured
+  # `discard_classes`
+  class DiscardErrorClass
+    ##
+    # @param middleware [#call] The next middleware to call
+    def initialize(middleware)
+      @middleware = middleware
+    end
+
+    ##
+    # @param report [Report]
+    def call(report)
+      ignore_error_class = report.raw_exceptions.any? do |ex|
```

Comment:
I know it's just a local variable name, but perhaps better refer to it as "discarding" rather than "ignoring" here?